slasher/db/kv/kv.go

```diff
@@ -76,6 +77,7 @@ func NewKVStore(dirPath string, cfg *Config) (*Store, error) {
 	if err := os.MkdirAll(dirPath, 0700); err != nil {
 		return nil, err
 	}
+	logrus.Infof(".db & validator keys dir is: %v", dirPath)
```

Comment:
This isn't needed, the slasher already logs out the datadir.

Reply:
So I keep the line for the .db?